front/components/assistant/conversation/ActionValidationProvider.tsx

```diff
@@ -220,7 +241,7 @@ export function ActionValidationProvider({
               {validationQueue.length > 0 && (
                 <div className="mt-2 text-sm font-medium text-info-900 dark:text-info-900-night">
                   {validationQueue.length} more request
-                  {validationQueue.length > 1 ? "s" : ""} in queue
```

Comment:
I know that was prior to the PR, but FYI we have a helper for that `pluralize` 😊

Reply:
okok 👍